src/SWP/Bundle/BridgeBundle/Resources/config/serializer/Model.Item.yml

```diff
@@ -5,3 +5,23 @@ SWP\Component\Bridge\Model\Item:
             expose: true
             serialized_name: body_html
             type: string
+        bodyText:
+            expose: true
+            serialized_name: body_text
+            type: string
+        description:
+            expose: true
+            serialized_name: description_text
+            type: string
+        usageTerms:
+            expose: true
+            type: string
+            serialized_name: usageterms
+        items:
```

Comment:
But You never know what's there - it's like item - have this same structure, and things like that can be more in future - i can't identify it as embed - here i know that items inside text items are embeds, but Item Item model is unaware about this.